app/uk/gov/hmrc/hmrcemailrenderer/templates/gms/gms_enrolment_confirmation.scala.html

```diff
@@ -0,0 +1,32 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.hmrcemailrenderer.templates.helpers.DateHelper
+@(params: Map[String, Any])
+
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "You successfully registered to get goods movement references") {
+
+<p style="margin: 0 0 30px; font-size: 19px;">Dear @{params("userName")}</p>
```

Comment:
there is a salutation template we should be using it and not our own custom one.

Reply:
That was failing because it was not assigned to our templateId which is now added in TemplateParams